folly/portability/Windows.h

```diff
@@ -37,6 +37,7 @@
 #include <direct.h> // nolint
 #endif
 
+#define NOMINMAX
```

Comment:
This is currently being defined by the CMake build system because it's a requirement for any external code using the library to do the same, otherwise any Windows header included before this will break the assumptions of this code.